Lib/sre_parse.py

```diff
@@ -260,6 +261,14 @@ def _class_escape(source, escape):
         elif c in DIGITS:
             raise error, "bogus escape: %s" % repr(escape)
         if len(escape) == 2:
+            if sys.py3kwarning and c in ASCIILETTERS:
+                import warnings
+                if c in 'Uu' and state.flags & SRE_FLAG_UNICODE:
+                    warnings.warn('unicode escape %s' % escape,
```

Comment:
Thank you @jwilk for finding all these bugs. Could you propose better error message?